src/web/event-notifications/event-notification-details/SlackNotificationDetails.jsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import {ReadOnlyFormGroup} from 'components/common';
+import {Well} from 'components/graylog';
+import styles from './SlackNotificationSummary.css';
+
+
+const SlackNotificationDetails = ({ notification }) => (
+  <>
+    <ReadOnlyFormGroup label="Webhook URL" value={notification.config.webhook_url} />
+    <ReadOnlyFormGroup label="Channel" value={notification.config.channel} />
+    <ReadOnlyFormGroup label="Custom Message Template "
+                       value={(
+                         <Well bsSize="small" className={styles.bodyPreview} >
+                           {notification.config.custom_message || <em>Empty body</em>}
+                         </Well>
+                       )} />
+  </>
+);
+
+SlackNotificationDetails.propTypes = {
+  notification: PropTypes.object.isRequired,
```

Comment:
I believe our front end engineers would be happier with a fully-specified shape here:
```suggestion
  notification: PropTypes.shape({
    config: PropTypes.shape({
      graylog_url: PropTypes.string,
      icon_emoji: PropTypes.string,
      icon_url: PropTypes.string,
      link_names: PropTypes.string,
      notify_channel: PropTypes.string,
      user_name: PropTypes.string,
      custom_message: PropTypes.string,
      channel: PropTypes.string,
      webhook_url: PropTypes.string,
      color: PropTypes.string,
    }).isRequired,
```

Reply:
@waab76 - I will look into this as a separate task.